hashes/src/hmac.rs

```diff
@@ -169,70 +169,70 @@ mod tests {
         use crate::{sha256, GeneralHash as _, Hash as _, HashEngine, Hmac, HmacEngine};
 
         #[derive(Clone)]
-        struct Test {
-            key: Vec<u8>,
-            input: Vec<u8>,
-            output: Vec<u8>,
+        struct Test<'a> {
+            key: &'a [u8],
+            input: &'a [u8],
+            output: &'a [u8],
```

Comment:
Can `output` be a fixed size array?

Reply:
As I can see could be, what about more tests in the future?